### PR TITLE
[ko] guide/keras/custom_callback

### PR DESCRIPTION
**Pull request by lamberta**
Migrate https://github.com/tensorflow/docs/pull/1214 by @ghdalsrl0708
CLA signed in that PR

### CODE REVIEW

File: site/ko/guide/keras/custom_callback.ipynb [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+        "  def __init__(self, patience=0):\n",
+        "    super(EarlyStoppingAtMinLoss, self).__init__()\n",
+        "\n",
+        "    self.patience = patience\n",
+        "\n",
+        "    # 최소 손실을 발생하고 가중치를 저장하는 최선의 무게.\n",
+        "    self.best_weights = None\n",
+        "\n",
+        "  def on_train_begin(self, logs=None):\n",
+        "    # 손실이 더 이상 최소가 아닐 때 기다려온 epoch의 수.\n",
```
[the remark this answers is not in the document]
Comment (lamberta) [reply site/ko/guide/keras/custom_callback.ipynb]: "epoch" in Korean do not translate list: https://github.com/tensorflow/docs-l10n/tree/master/site/ko#korean-translation-guide

File: site/ko/guide/keras/custom_callback.ipynb [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0c_TYhQOUe1j"
+      },
+      "source": [
+        "## 케라스 callbacks 도입\n",
```
[the remark this answers is not in the document]
Comment (lamberta) [reply site/ko/guide/keras/custom_callback.ipynb]: "callback" in Korean do not translate list: https://github.com/tensorflow/docs-l10n/tree/master/site/ko#korean-translation-guide

File: site/ko/guide/keras/custom_callback.ipynb [hunk 3]
```diff
@@ -0,0 +1,562 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "fIy5JKMlZNmh"
+      },
+      "source": [
+        "##callbacks을 수행하는 모델 방법\n",
+        "사용자는 다음 텐서플로우.케랏.모델 방법을 따라 callback 리스트를 제공합니다:\n",
+        "#### [`fit()`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/Model#fit), [`fit_generator()`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/Model#fit_generator)\n",
+        "일정한 수의 epochs(데이터 집합에 대한 반복, 또는 파이썬 생성기에 의한 batch별 데이터 산출)에 대한 모델을 추적합니다..\n",
```
[the remark this answers is not in the document]
Comment (lamberta) [reply site/ko/guide/keras/custom_callback.ipynb]: "epoch" in Korean do not translate list: https://github.com/tensorflow/docs-l10n/tree/master/site/ko#korean-translation-guide

File: site/ko/guide/keras/custom_callback.ipynb [hunk 4]
```diff
@@ -0,0 +1,562 @@
...
+        "\n",
+        "\n",
+        "### 교육/테스트/예측을 위한 일반적인 방법\n",
+        "트레이닝, 테스트 및 예측을 위해 재정의할 수 있는 방법은 다음과 같습니다.\n",
+        "#### `on_(train|test|predict)_begin(self, logs=None)`\n",
+        "'적합성'/'평가'/'예측'의 시작을 호출합니다.\n",
+        "#### `on_(train|test|predict)_end(self, logs=None)`\n",
+        "'적합성'/'평가'/'예측'의 끝을 호출합니다.\n",
+        "#### `on_(train|test|predict)_batch_begin(self, batch, logs=None)`\n",
+        "트레이닝/시험/예측이 진행되는 동안 batch의 진행 직전에 호출됩니다. 이 메소드 내에서, 일지는 batch의 현재 수와 사이즈를 나타내는 자유로운 크기의 키로 받아 써집니다.\n",
```
[the remark this answers is not in the document]
Comment (lamberta) [reply site/ko/guide/keras/custom_callback.ipynb]: "batch" in Korean do not translate list: https://github.com/tensorflow/docs-l10n/tree/master/site/ko#korean-translation-guide

**Comment by jaketae** [comment]
@lamberta Thank you for the quick review. I was aware of the list, but I was under the impression that "do no translate" meant "do not use a Korean word to substitute the original." 콜백, pronounced as something like "kol-beck," is a transliteration and is thus technically not a translation; the same goes for 에폭, which is pronounced as "ae-pok." Do you think we need to use original English words instead of transliterations? Thanks again.

**Comment by lamberta** [comment]
Ah. Well, that sounds like a matter of style for the Korean translation community to decide. How is this handled in the other guides? (Let's say something that was translated a while back) cc: @rickiepark

**Comment by rickiepark** [comment]
Thanks @lamberta, @jaketae 
All english words is translated except abbreviations, because Korean has the loanword orthography. Please check this link (http://kornorms.korean.go.kr/regltn/regltnView.do?regltn_code=0003&regltn_no=425#a425).
Thank you! :)

**Comment by jaketae** [comment]
@rickiepark Thank you for the heads up! 

@lamberta Seems like transliteration is the accepted practice here, so I'll stick with the suggestions I've made. Thank you for double checking though! Perhaps I'll edit the README to help clarify what "do not translate" entails, possibly along with the link @rickiepark posted in his comments.

**Pull request closed without merging**